app/src/main/java/org/greenstand/android/TreeTracker/signup/CredentialEntryView.kt

```diff
@@ -115,7 +115,7 @@ fun CredentialEntryView(viewModel: SignupViewModel, state: SignUpState) {
                     onValueChange = { updatedEmail -> viewModel.updateEmail(updatedEmail) },
                     placeholder = { Text(text = stringResource(id = R.string.email_placeholder), color = Color.White) },
                     keyboardOptions = KeyboardOptions(
-                        keyboardType = KeyboardType.Email,
+                        keyboardType = KeyboardType.Password,
```

Comment:
I think we should leave this as email since it will make it much easier for people to type an email who have not used a phone before. Looking at solutions online, there doesn't seem to be a fix that works for all phones. Of the fixes I saw (while leaving it as email), they only work in XML, not compose.

Like Natalie said, different keyboards will show differently. I think the actual solution to this might be the user disabling suggestions on their keyboard.

I'd suggest turning off autocorrect for the keyboard.

Reply:
Yes I thought as much, so we can close this PR without merging it.